idsvr/templates/config-backup.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if .Values.curity.config.backup -}}
+{{- if and .Values.curity.config.backup -}}
```

Comment:
missing second part of `and`

Reply:
I added some changes (love) in commit below.